public/i18n/qqq.json

```
@@ -25,8 +25,8 @@
     "column-external-source": "External source column on mismatches table",
     "no-mismatches-found-message": "A message when no mismatches have been found for the provided Item Ids",
     "review-status-pending": "Label for pending review status. Used as placeholder in the review decision dropdown",
-    "review-status-wikidata": "Label that indicates that the data mismatch is on Wikidata. Used as an option in the review decision dropdown",
-    "review-status-external": "Label that indicates that the data mismatch is on the external database. Used as an option in the review decision dropdown",
+    "review-status-wikidata": "Label that indicates that the mismatch is due to wrong data on Wikidata. Used as an option in the review decision dropdown",
+    "review-status-external": "Label that indicates that the mismatch is due to wrong sata on the external database. Used as an option in the review decision dropdown",
```

```suggestion
    "review-status-external": "Label that indicates that the mismatch is due to wrong data on the external database. Used as an option in the review decision dropdown",
```

Data vs. SATA :wink: